src/commands/docs/generateDocs.ts

```diff
@@ -5,41 +5,52 @@ import ora from 'ora'
 
 import { isWindows } from '../../utils/command'
 import { createFolder, deleteFolder, folderExists } from '../../utils/file'
-import { findTargetInConfiguration, getLocalConfig } from '../../utils/config'
+import { getLocalConfig } from '../../utils/config'
 import { getConstants } from '../../constants'
 
 import { getFoldersForDocs } from './internal/getFoldersForDocs'
-import { getFoldersForDocsAllTargets } from './internal/getFoldersForDocsAllTargets'
+import { createDotFiles } from './internal/createDotFiles'
+import { getDocConfig } from './internal/getDocConfig'
 
 /**
  * Generates documentation(Doxygen)
  * By default the docs will be at 'sasjsbuild/docs' folder
  * If a target is supplied, generates docs only for the SAS programs / macros / jobs / services in that target (and the root).
  * If no target is supplied, generates for all sas programs/ macros / jobs / services.
  * @param {string} targetName- the name of the target to be specific for docs.
- * @param {string} outDirectory- the name of the output folder, picks from sasjsconfig.docConfig if present.
+ * @param {string} outDirectoryInput- the name of the output folder, picks from sasjsconfig.docConfig if present.
```

Comment:
Why is in `outDirectoryInput` word `input` more descriptive?

Reply:
I agree. I think outDirectory was more descriptive than outDirectoryInput.

Reply:
Updated